draft-demarco-oauth-status-attestations.md

```diff
@@ -361,6 +366,32 @@ Below a non-normative example of an HTTP Response with an error.
   }
 ~~~
 
+Since the Status Assertion Request is an array of strings, if one or more status
+assertion request are made for a Digital Credential with errors,
+the response status code will be always `200` including the parameters as specified in  the OAuth 2.0 RFC
+[Section 5.2](https://tools.ietf.org/html/rfc6749#section-5.2).
+
+In addition to the error codes referenced in {{RFC6749}}, the following error codes
+MUST also be supported:
+
+| Error Parameter Value | Description | Reference |
+| --- | --- | --- |
+| **credential_revoked** | The Digital Credential is revoked. The reason of revocation MUST be provided in the error_description field. | this specification |
+| **credential_updated** | One or more attributes contained in the Digital Credential are changed. The error_description field MUST contain a human-readable text describing the general parameters updated without specifying each one. | this specification |
+| **credential_invalid** | The Digital Credential is invalid. The error_description field MUST contain the reason of invalidation. | this specification |
+
+Below a non-normative example of an HTTP Response with one or more Status Assertion Requests with error.
+
+~~~
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+    "status_assertion_responses": ["${base64url(json({typ: status-assertion-error+jwt, ...}))}.payload.signature",
+"${base64url(json({typ: status-assertion-error+cwt, ...}))}.payload.signature", ... ]
```

Comment:
```suggestion
"${base64url(json({typ: status-assertion-error+cwt, ...}))}.payload", ... ]
```

@OR13 @fmarino-ipzs WDYT if the status assertion errors may be unsigned, therefore using the alg `none`?

Reply:
In my opinion, status assertion errors cannot be unsigned because the description parameter requires the reason for revocation to be specified. Depending on the level of detail in the "error_description" field, this could lead to privacy issues.

Reply:
therefore we must say that the error description message MUST not contain any mention to personal data or credential types

because it's in the scope of this specification that the status assertion must not, in any case, disclose any personal data o credential data outside of an opaque value

Reply:
I generally don't think any specification should be encouraging the use of alg none, in the year 2024.
If its not forbidden explicitly, it should certainly not be encouraged.